site/ko/blog/new-in-devtools-96/index.md

```diff
@@ -47,28 +47,48 @@ draft: true
 Chromium issue: [1254557](https://crbug.com/1254557)
 
 
-<!-- ## Rendering tab updates  -->
+<!-- ## Restored and improved CSS length edit and copy experince {: #length } -->
 
-<!-- ### Emulate the CSS prefers-contrast media feature {: #prefers-contrast } -->
```

Comment:
I added a new section here due to we merged some fixes in M96. Please help to translate. Thanks!

Reply:
Added new translation for the new section